kuma/wiki/helpers.py

```diff
@@ -232,3 +233,24 @@ def wiki_url(path):
     Look for a wiki page in the current locale, or default to given path
     """
     return reverse('wiki.document', args=[path])
+
+
+def url_list(urls):
+    js_list = '[' + ', '.join('"%s"' % url for url in urls) + ']'
+    return jinja2.Markup(js_list)
+
+
+@register.function
+def js_bundle_urls(bundle, debug=None):
+    """
+    Return a list of the URLS for a minified and bundled JS file
```

Comment:
I'm expanding the doc strings with some examples to make it clearer, and changing the names to ``js_url_array`` and ``css_url_array``.